subsys/bootloader/bl_storage/nrf_nv_counters.c

```diff
@@ -24,22 +24,22 @@ fih_int boot_nv_security_counter_get(uint32_t image_id, fih_int *security_cnt)
 	uint16_t cur_sec_cnt;
 
 	if (security_cnt == NULL) {
-		return FIH_FAILURE;
+		FIH_RET(FIH_FAILURE);
```

Comment:
Is this file used for both B0/NSIB and mcuboot?

Reply:
no, this is the implementation used only by MCUboot. B0 has access to the counters through this:
https://github.com/nrfconnect/sdk-nrf/blob/343ba1e878ace99992a299cf58966d536bec5c36/subsys/bootloader/bl_storage/bl_storage.c#L241